composer.json

```diff
@@ -18,7 +18,7 @@
         "source": "https://github.com/cakephp/cakephp"
     },
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.6.0",
```

Comment:
@markstory that's easy we don't use a custom zip file anymore but it's downloaded from php website. I can do it if you want